web_widget_image_webcam/static/src/xml/web_widget_image_webcam.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<template>
+    <t t-extend="FieldBinaryImage">
+        <t t-jquery=".oe_form_field_image_controls" t-operation="after">
```

Comment:
@Yajo , Yes There is already icon bar exist. but when I put webcam icon in current icon-bar then its looks like this on runbot. [There is one more icon(download) from another module.]

![demo_hr_page](https://cloud.githubusercontent.com/assets/8414798/18607916/5392cf88-7cf8-11e6-87b3-6a1e32aa16af.png)

So, Is it good to put like this (As in screen-shot), Or bottom-right-corner of image (Already done).

Need Suggestions...!!

--
Thanks.